lib/ricohapi/mstorage/client.rb

```diff
@@ -21,6 +21,7 @@ def initialize(access_token)
       def list(params = {})
         params.reject! do |k, v|
           ![:after, :before, :limit, :filter].include? k.to_sym
+          v.nil?
```

Comment:
I guess we should fix codes around L32~34 to remove keys with nil value.
I tried a simpler method in irb.
![2016-07-25 15 03 30](https://cloud.githubusercontent.com/assets/16602330/17091837/02f40d66-5279-11e6-8e68-ccc27c839285.png)